README.md

```diff
@@ -109,6 +109,14 @@ commit your own `.semgrepignore`.
 Note that `.semgrepignore` is picked up only by the action,
 and will not be honored when running `semgrep` manually.
 
+### Silent Mode
```

Comment:
@underyx @pabloest During testing today "silent" proved to be a confusing term our user and it wasn't clear from the title if meant it also succeeded if there were errors. Two ideas:

1. This feature could be renamed, perhaps to "audit" or some other indicator of being run in the background without disturbing the dev flow
2. Not configuring this in the workflow file. As is it reads as a duplicate capability to running a policy that doesn't block. I worry about user's trying to debug the lack of a breaking main/develop build when their policy says to block, but this overrides that behavior (and was set long ago when they first added Semgrep and have since forgotten). Could this capability be downscoped for now and considered as part of @chmccreery's work or a corollary to @brendongo's fail open feature?